store.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+    <meta name="description" content="Internet for all devices" />
+    <meta name="keywords" content="Wifi,Internet,productivity,karma" />
+    <title>Karma - Store</title>
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,300" rel="stylesheet" type="text/css" />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <body>
+    <!-- Add your HTML markup here -->
+    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <!-- All the images you need are in the 'img' folder -->
+
+    <header class="header">
+      <a href="index.html"> <img src="img/karma-logo.svg" alt="karma-logo" /> </a>
+      <nav class="nav-menu">
+        <li><a href="index.html">Meet Karma</a></li>
+        <li><a href="#">How it Works</a></li>
+        <li><a class="current-menu-item" href="#">Store</a></li>
+        <li><a href="#">Blog</a></li>
+        <li><a href="#">Help</a></li>
+        <li><a href="#">Login</a></li>
+      </nav>
+    </header>
+    <main>
+      <!-- Store section -->
+      <section class="store">
+        <div class="store-main">
+          <h2>Order your Karma wifi device today!</h2>
+          <form class="flex-column">
+            <div class="form-grid">
+              <div class="col-3 flex-column">
+                <label for="name">First name *</label>
+                <input type="text" id="name" />
+              </div>
+              <div class="col-3 flex-column">
+                <label for="last-name">Last name *</label>
+                <input type="text" id="last-name" />
+              </div>
+              <div class="col-6 flex-column">
+                <label for="address">Address *</label>
+                <input type="text" id="address" />
+              </div>
+              <div class="col-6 flex-column">
+                <label for="address-2">Address 2 *</label>
+                <input type="text" id="address-2" />
+              </div>
+
+              <div class="col-4 flex-column">
+                <label for="city">City *</label>
+                <select id="city" name="city">
+                  <option value="" disabled selected>Select your city...</option>
```

Comment:
I really like the use of disabled and selected together, it's a nice touch that you can't select the empty option